test/test_podman_baseline.sh

```diff
@@ -7,20 +7,30 @@
 #
 # To run this command:
 #
+# /bin/bash -v test_podman_baseline.sh -d # Install and then deinstall Docker
 # /bin/bash -v test_podman_baseline.sh -e # Stop on error
 # /bin/bash -v test_podman_baseline.sh    # Continue on error
 
 #######
-# See if we want to stop on errors or not.
+# See if we want to stop on errors and/or install and then remove Docker.
 #######
 showerror=0
-while getopts "e" opt; do
+installdocker=0
+while getopts "de" opt; do
     case "$opt" in
+    d) installdocker=1
```

Comment:
I personally would prefer a test for docker and or an active daemon rather than a switch.  just my way ...

Reply:
Yeah, I just went with the KISS method.